x-pack/plugins/secops/public/components/flyout/index.test.tsx

```diff
@@ -351,15 +351,16 @@ describe('Flyout', () => {
       const openMock = jest.fn();
       const wrapper = mount(
         <ReduxStoreProvider store={store}>
-          <DragDropContextWrapper>
-            <FlyoutButton
-              dataProviders={mockDataProviders}
-              show={true}
-              theme="dark"
-              timelineId="test"
-              onOpen={openMock}
-            />{' '}
-          </DragDropContextWrapper>
+          <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
+            <DragDropContextWrapper>
+              <FlyoutButton
+                dataProviders={mockDataProviders}
+                show={true}
+                timelineId="test"
+                onOpen={openMock}
+              />{' '}
```

Comment:
I know you didn't add this ugly weird `{' '}`, I'm pretty sure that was me. However, pointing out that it's not needed and if you want you can delete `{' '}`. I think that's an accident on my part at some point where my IDE added that funny character and prettier said, "sure thing, let's make it 💄 pretty".

I removed it in my env. and all tests still work ok, if you want to get rid of it at this point.